lib/JSON/Validator.pm

```diff
@@ -1291,7 +1291,10 @@ See L</Bundled specifications> for more details.
   my $jv = $jv->formats(\%hash);
 
 Holds a hash-ref, where the keys are supported JSON type "formats", and
-the values holds a code block which can validate a given format. A code
+the values holds a code block which can validate a value against a given
```

Comment:
side observation, to @jhthorsen -- it doesn't look like there are any tests in the distribution for the user adding his own formats to the validator -- e.g. `$jv->formats->{my_custom_format} = sub { ... }`. I didn't realize this was supported until I saw this PR!

Reply:
@karenetheridge: If that didn't work, then none of the format tests would work. I'm not encouraging adding custom formats though, since I'm not sure if that will actually result in a portable schema.